### DISCUSSION TITLE
What is the purpose of the APIName/RecordableAPIName?

### DISCUSSION THREAD

**Discussion by BohdanHorban** [ansys/pymechanical#615] [Q&A]
Hello all, using dir() I can get all available object attributes. I wonder what is the purpose of APIName and RecordableAPIName?
![ApiName](https://github.com/ansys/pymechanical/assets/154509349/cf6a9c96-b975-4177-bd4e-704145813590)

**Comment by dipinknair** [comment]
This is related to Mechanical rather than pymechanical. You can direct this question here https://discuss.ansys.com/categories/structures
